.pipelines/networkobservability/pipeline.yaml

```diff
@@ -0,0 +1,145 @@
+pr: none
+trigger: none
+
+schedules:
+- cron: '0 0 * * *'
```

Comment:
I like that we are controlling the scheduling from the yaml. Just be advised that changes within the AZP UI will override what you have here

Reply:
That is a good call out. In some cases I could see it useful that we have that ability.